bin/tictactoe

```diff
@@ -1,3 +1,16 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+require 'pry'
+
+puts "Greetings!"
+
+cli = Cli.new
```

Comment:
Great job on abstracting this class, but all of the code should be located in the cli. All that should be on this page is `Cli.new.setup_game`